MyRWTutorial.xcodeproj/xcshareddata/IDETemplateMacros.plist

```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
 	<key>FILEHEADER</key>
-	<string>/// Copyright (c) 2017 Razeware LLC
+	<string>/// Copyright (c) ___YEAR___ Razeware LLC
```

Comment:
We should also remove the first 2 `//` from this line. Xcode inserts them automatically (thanks Xcode....) so in the current project any new files get `/////`.